docs/public-networks/reference/evm-tool.md

```diff
@@ -153,7 +155,25 @@ The amount is not reduced to cover intrinsic cost and gas fees.
     --json=true
     ```
 
-Provide an operation-by-operation trace of the command in JSON when set to true.
+When set to `true`, provides an operation-by-operation trace of the command in JSON.
+The default is `false`.
+
+### `json-alloc`
+
+=== "Syntax"
+
+    ```bash
+    --json-alloc=<boolean>
```

Comment:
Just `--json-alloc`.

"When the option is set a json summary of the the post-execution world state and allocations will be printed"